.vib/apache/vib-publish.json

```
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/apache"
         },
-        "runtime_parameters": "InNlcnZpY2UiOgogICJwb3J0IjogODAKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIi",
+        "runtime_parameters": "InNlcnZpY2UiOgogICJwb3J0cyI6CiAgICAiaHR0cCI6IDgwCiAgICAiaHR0cHMiOiA0NDMKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIiCiJob3N0QWxpYXNlcyI6CiAgLSAiaXAiOiAxMjcuMC4wLjEKICAgICJob3N0bmFtZXMiOgogICAgICAgIC0gInN0YXR1cy5sb2NhbGhvc3Qi",
```

You are missing some values here:

1) While you can't change the default `containerPorts` (because the chart fails to do so), we should explicitly set them here. In the future when this is fixed, we will just need to update the value.

2) The HTTPs port for the LoadBalancer can be modified 😁. The main app's port should always be `80`, but other ports can be modified to check this customization works.

3) We are also adding `podSecurityContext` and `containerSecurityContext` by convention now.

```suggestion
        "runtime_parameters": "c2VydmljZToKICB0eXBlOiBMb2FkQmFsYW5jZXIKICBwb3J0czoKICAgIGh0dHA6IDgwCiAgICBodHRwczogNDQ0CmNvbnRhaW5lclBvcnRzOgogIGh0dHA6IDgwODAKICBodHRwczogODQ0Mwpob3N0QWxpYXNlczoKICAtIGlwOiAiMTI3LjAuMC4xIgogICAgaG9zdG5hbWVzOgogICAgICAtICJzdGF0dXMubG9jYWxob3N0Igpwb2RTZWN1cml0eUNvbnRleHQ6CiAgZW5hYmxlZDogdHJ1ZQogIGZzR3JvdXA6IDEwMDIKY29udGFpbmVyU2VjdXJpdHlDb250ZXh0OgogIGVuYWJsZWQ6IHRydWUKICBydW5Bc1VzZXI6IDEwMDI=",
```